include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -142,3 +142,9 @@ static const char* const kOrtSessionOptionsConfigForceSpinningStop = "session.fo
 // "0": in some cases warnings will be logged but processing will continue. The default.
 // May be useful to expose bugs in models.
 static const char* const kOrtSessionOptionsConfigStrictShapeTypeInference = "session.strict_shape_type_inference";
+
+// Configuration entry prefix for custom operators. Entry keys for custom operators are built
+// at runtime with the following structure:
+// "custom_op.[CustomOpName].[Keyname]"
+// Ex: "custom_op.MyCustomOp.device_type"
+static const char* const kOrtSessionOptionsConfigCustomOpConfigPrefix = "custom_op.";
```

Comment:
Should this be called ```proxyep.``` since these are really configs for the proxy EP?

Reply:
I recall some confusion/ambiguity around calling this an "EP". Do we want to call this a "proxy runtime" or "proxy custom op"?